akka-stream-tck/src/test/scala/akka/stream/tck/ActorPublisherTest.scala

```diff
@@ -40,6 +40,8 @@ object ActorPublisherTest {
 
 class ActorPublisherTest extends AkkaPublisherVerification[Int] {
 
+  override lazy val system = createActorSystem()
```

Comment:
which constructor? I removed ActorSystem creation from all constructors because TestNG creates 2 instances of the test class and one of them is not used and @AfterClass is not invoked on one of
the instances, i.e. the system is not shutdown.

It is also difficult to find a good ActorSystem name when creating from constructor.